src/test/java/picard/vcf/UpdateVcfSequenceDictionaryTest.java

```diff
@@ -28,38 +28,61 @@
 import org.testng.Assert;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.Test;
+import org.testng.annotations.DataProvider;
 
+import java.io.BufferedOutputStream;
 import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
 
 /**
  * @author George Grant
  */
 public class UpdateVcfSequenceDictionaryTest {
     private static final File TEST_DATA_PATH = new File("testdata/picard/vcf/");
     private static final File OUTPUT_DATA_PATH = IOUtil.createTempDir("UpdateVcfSequenceDictionaryTest", null);
+    private static final File STD_OUT_FILE = new File(OUTPUT_DATA_PATH, "stdout.vcf");
```

Comment:
I think you should be calling `System.setOut(PrintStream out)` from the test and not modify the code in UpdateVcfSequenceDictionary. 

so the test would be something like 
```java
File STD_OUT_FILE = new File(OUTPUT_DATA_PATH, "stdout.vcf");
Stream stdout = new OutputStream(STD_OUT_FILE);
System.setOut(stdout);

//run test

//close stream if needed

//examine file.
```